lab1/lab1.rst

```diff
@@ -0,0 +1,412 @@
+.. raw:: latex
```

Comment:
Let's remove this. Also, could you add ":Author:" (or, more correctly, ":Editor:" line at the beginning?